localization/english/CWP/CWP_buildings_l_english.yml

```
@@ -120,25 +120,53 @@
 
   # Building Groups
   bg_housing: "Housing"
+  bg_slums: "Slums"
   bg_air: "Air Force Bases"
   bg_army: "Army Bases"
-  bg_arts: "Arts industry"
+  bg_navy: "Navy Bases"
+  bg_wmd: "WMD Bases"
+  bg_nuclear_weapons: "Nuclear Silos"
+  bg_chemical_weapons: "Chemical Silos"
+  bg_biological_weapons: "Biological Silos"
+  bg_arts: "Arts Industries"
   bg_bauxite_mining: "Bauxite Mines"
   bg_copper_mining: "Copper Mines"
-  bg_financial_services: "Financial services industry"
+  bg_financial_services: "Financial Services Industries"
   bg_fossil_fuels: "Fossil Fuels"
   bg_humanities_research: "Humanities Research Institutions"
-  bg_media: "Media industry"
+  bg_media: "Media Industries"
   bg_military_research: "Military Research Institutions"
-  bg_nuclear_power: "Nuclear power industry"
+  bg_nuclear_power: "Nuclear Power Industries"
   bg_nuclear_research: "Nuclear Research Institutions"
   bg_phosphorus_mining: "Phosphorus Mines"
   bg_rare_earth_elements_mining: "Rare Earths Mines"
   bg_technology: "Technological Institutions"
-  bg_telecommunications: "Telecommunications industry"
-  bg_tourism: "Tourism industry"
+  bg_telecommunications: "Telecommunications Industries"
```

syntax error: double plural.
```suggestion
  bg_telecommunications: "Telecommunication Industries"
```

---

fixed